apps/send-func/src/adapters/functions/aar-notifications.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  HttpHandler,
+  HttpRequest,
+  HttpResponseInit,
+  InvocationContext,
+} from "@azure/functions";
+import { lollipopHeadersSchema } from "io-messages-common/adapters/lollipop/definitions/lollipop-headers";
+import { lollipopExtraInputsCtxKey } from "io-messages-common/adapters/lollipop/lollipop-middleware";
+
+import {
+  iunSchema,
+  mandateIdSchema,
+  problemJsonSchema,
+  sendHeadersSchema,
+} from "../send/definitions.js";
+import NotificationClient, {
+  NotificationClientError,
+} from "../send/notification.js";
+
+export const getNotification =
+  (
+    notificationClient: NotificationClient,
+    uatNotificationClient: NotificationClient,
+  ): HttpHandler =>
+  async (
+    request: HttpRequest,
+    context: InvocationContext,
+  ): Promise<HttpResponseInit> => {
+    try {
+      const isTest = request.query.get("isTest") === "true";
+      const client = isTest ? uatNotificationClient : notificationClient;
+
+      const lollipopHeaders = lollipopHeadersSchema.safeParse(
+        context.extraInputs.get(lollipopExtraInputsCtxKey),
+      );
+
+      const sendHeaders = sendHeadersSchema.safeParse({
+        "x-pagopa-cx-taxid": request.headers.get("x-pagopa-cx-taxid"),
+        "x-pagopa-pn-io-src":
+          request.headers.get("x-pagopa-pn-io-src") || undefined,
+        ...lollipopHeaders.data,
+      });
+
+      if (!lollipopHeaders.success || !sendHeaders.success) {
```

Comment:
```suggestion
      if (!sendHeaders.success) {
```

There is no need to check for lollipopHeaders.success here, sendHeaders.success would be false if lollipopHeaders.success is

Reply:
731d33f38a95d6822337f0628c242b01b3833edb